ams/server/src/main/java/com/netease/arctic/server/dashboard/PaimonTableDescriptor.java

```diff
@@ -62,6 +71,17 @@
  * Descriptor for Paimon format tables.
  */
 public class PaimonTableDescriptor implements FormatTableDescriptor {
+
+  private final Executor executor;
```

Comment:
We have to close the Executor after HA since it's a fixed thread pool.

Reply:
It shouldn't need to be closed. If a new object is created, the thread pool will be garbage collected along with the old object.